apps/src/blockly/addons/contextMenu.js

```diff
@@ -184,6 +185,36 @@ const registerUnshadow = function() {
   GoogleBlockly.ContextMenuRegistry.registry.register(unshadowOption);
 };
 
+const registerKeyboardNavigation = function() {
+  const keyboardNavigationOption = {
+    displayText: function(scope) {
+      const displayText = scope.workspace.keyboardAccessibilityMode
+        ? msg.blocklyKBNavOff()
+        : msg.blocklyKBNavOn();
+      return displayText;
+    },
+    preconditionFn: function() {
+      if (Blockly.navigationController) {
+        return 'enabled';
+      } else {
+        console.log('navigationController not found');
+        return 'hidden';
+      }
+    },
+    callback: function(scope) {
+      if (scope.workspace.keyboardAccessibilityMode) {
+        Blockly.navigationController.disable(scope.workspace);
+      } else {
+        Blockly.navigationController.enable(scope.workspace);
+      }
```

Comment:
super nit / subjective comment: a ternary might be clearer here

Reply:
Oh, good call. Believe it or not, I think I've only ever used ternary to produce a value (like `displayText` above), and hadn't ever considered using them to call functions like these that don't return anything.